File: test/end-to-end/Dictionary/dictionary.java
-
// Dictionary types
HashMap<string, int> foo = new HashMap<string, int>();
HashMap<string, HashMap<string, int>> bar = new HashMap<string, new HashMap<string, int>>();

// Indices
foo["baz"] = 7;
int qux = foo["baz"];

// In-place initialization
HashMap<string, int> aaa = new HashMap<string, int>() {{
    put("bbb", 1);
    put("ccc", 2);
    put("ddd", 3);
}};
-

File: test/end-to-end/Dictionary/dictionary.cs
-
// Dictionary types
Dictionary<string, int> foo = new Dictionary<string, int>();
Dictionary<string, Dictionary<string, int>> bar = new Dictionary<string, new Dictionary<string, int>>();

// Indices
foo["baz"] = 7;
int qux = foo["baz"];

// In-place initialization
Dictionary<string, int> aaa = new Dictionary<string, int>
{
    { "bbb", 1 },
    { "ccc", 2 },
    { "ddd", 3 }
};
-